### DISCUSSION TITLE
Divergence in combustion behavior between FDS 6.8 & 6.9 - Particle Fuel Model

### DISCUSSION THREAD

**Discussion by sebastianmezaurotadze** [firemodels/fds#13236] [Q&A] [answered]
I'm running a simulation where I ignite some vegetation particles with a radiative heater and then measure the HRR. Running the same FDS code yields significantly different HRR results in 6.8 & 6.9: Particles ignite significantly later in 6.9 and burn at a higher peak HRR.

Were there any updates in particle modeling that could have such a large effect on the physics of the simulation? How can I be sure combustion and optical properties of the particles are modeled accurately?

**Comment by ericvmueller** [comment, marked as answer]
Without knowing more about your case it is difficult to say, as predicting ignition and burning is research which involves the coupling of many parts of the code (including but not limited to particles) which may have changed. 
I would recommend looking through the latest version of the various guides for relevant examples around vegetation (eg. [verification cases here](https://github.com/firemodels/fds/tree/master/Verification/WUI)) and particles in general (eg. [verification cases here](https://github.com/firemodels/fds/tree/master/Verification/Sprinklers_and_Sprays)) to understand how the code currently performs for simple tests relevant to your scenario.
